Content.Shared/Medical/Organs/Components/HeartComponent.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Content.Shared.FixedPoint;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Medical.Organs.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class HeartComponent : Component
+{
+
+    //TODO: implement fibrillation :^)
+
+    /// <summary>
+    /// Is this heart currently beating?
+    /// For mapped organs this should be set to false, unless you want the heart to beat while
+    /// it's sitting on the ground. Fun fact: the heart contains its own neurons and will keep beating even after death or
+    /// removal from the body as long as it is supplied with oxygen.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public bool IsBeating = true;
+
+
+    /// <summary>
+    /// How much blood does this heart pump per cycle
+    /// </summary>
+    [DataField, AutoNetworkedField]//TODO: Required
+    public FixedPoint2 PumpVolume = 30;
+
+    /// <summary>
+    /// The resting heartrate for this organ
+    /// </summary>
+    [DataField, AutoNetworkedField] //TODO: required
+    public FixedPoint2 RestingRate = 90;
+
+    /// <summary>
+    /// The current heart rate. If this is less than 0, it will be set to the resting rate on map init.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public FixedPoint2 CurrentRate = -1;
+
+    /// <summary>
+    /// The maximum rate that this heart will safely beat at.
+    /// </summary>
+    [DataField(required:true), AutoNetworkedField]
+    public FixedPoint2 MaximumRate = 190;
```

Comment:
Just a tiny suggestion, Maximum rate should be given by 220 - player age in years. (Nitpicking i know but I figured i'd write it still)

Reply:
> Just a tiny suggestion, Maximum rate should be given by 220 - player age in years. (Nitpicking i know but I figured i'd write it still)

Why?

Reply:
That's the formula I learned in med school for calculating maximum heart rate for a given person. It considers age as a factor, which can sort of simulate older patients being more prone to heart attack vs. younger patients. Atrial fibrillation for example is very uncommon and noticeable to young adults experiencing it, on the contrary senior residents can be in a situation where more than half of their heartbeats are skipped or incomplete. I understand if Jezithyr finds it superfluous.

Reply:
sounds cool

Reply:
> Just a tiny suggestion, Maximum rate should be given by 220 - player age in years. (Nitpicking i know but I figured i'd write it still)

Yeah I was originally going to use that formula but I want respiratory parameters to be configurable per species. I also wanted to simplify things a little bit for the sake of players, so that they didn't need to worry about patient age. (Atleast in a first version)